azure-pipelines.yml

```diff
@@ -64,6 +64,22 @@ jobs:
         # - release/3.1.3xx
         - 3.0
         - release/5.0.1xx
+
+  # Ubuntu 1804 x64 SDK scenario benchmarks
+  - template: /eng/performance/scenarios.yml
+    parameters:
+      osName: ubuntu
+      osVersion: 1804
+      kind: scenarios
+      architecture: x64
+      pool: Hosted Ubuntu 1604
+      queue: Ubuntu.1804.Amd64.Open
+      container: ubuntu_x64_build_container
+      projectFile: scenarios.proj
+      channels:
+        - master
+        - release/5.0.1xx
+        # - release/3.1.3xx
```

Comment:
Do we need to turn this on for physical machines also? This just the VM queue, yes?

Reply:
Thanks for the reminder! added this in the new push.